### PR TITLE
fix(flutter_bloc): BlocListener, BlocProvider missing child assertion

### PR DESCRIPTION
**Pull request by felangel**
## Status

**READY**

## Breaking Changes

NO

## Description

- fix(flutter_bloc): BlocListener, BlocProvider missing child assertion

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [X] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: packages/flutter_bloc/test/bloc_listener_test.dart [hunk 1]
```diff
@@ -72,6 +72,23 @@ class _MyAppState extends State<MyApp> {
...
+        BlocListener<CounterCubit, int>(
+          bloc: CounterCubit(),
+          listener: (context, state) {},
+        ),
+      );
+      expect(
+        tester.takeException(),
+        isA<AssertionError>().having((e) => e.message, 'message', expected),
+      );
+    });
```
Comment (jeroen-meijer) [inline packages/flutter_bloc/test/bloc_listener_test.dart:90]:
```suggestion
    },
    );
```

and format

Same goes for other tests, though I'm not sure what you're styling convention is here so feel free to ignore! 😄

**Review by jeroen-meijer — APPROVED** [review]
Minor styling comments. Other than that; 👌🏻 :shipit:

File: packages/flutter_bloc/test/bloc_listener_test.dart (hunk 1 above)
Comment (felangel) [reply packages/flutter_bloc/test/bloc_listener_test.dart:90]: For internal consistency I prefer to leave the test format as is but thanks for the review, really appreciate it!

**Review by marcossevilla — APPROVED** [review]
LGTM 🚀

**Pull request merged**